composer.json

```diff
@@ -11,16 +11,16 @@
     "require": {
         "php": "^8.0",
         "ext-json": "*",
-        "doctrine/doctrine-module": "^4.4.0",
-        "doctrine/doctrine-orm-module": "^4.2.1",
-        "laminas/laminas-cache": "^3.1.3",
         "laminas/laminas-db": "^2.9",
         "laminas/laminas-dependency-plugin": "^2",
         "laminas/laminas-eventmanager": "^3.2",
         "laminas/laminas-http": "^2.8",
         "laminas/laminas-log": "^2.10",
         "laminas/laminas-servicemanager": "^3.10",
-        "laminas/laminas-stdlib": "^3.2"
+        "laminas/laminas-stdlib": "^3.2",
+        "doctrine/doctrine-module": "^5.3",
+        "doctrine/doctrine-orm-module": "^5.3",
```

Comment:
Are we going to test this version bump by just updating mot-logger in MTS first as part of BL-15027? There are some breaking changes flagged in the release docs for this major bump

Reply:
Yeah I'm hitting some version conflicts with mot-logger on the composer.json which is blocking me from continuing on with the repoint ticket. Do you know what they breaking changes are? Currently doing some manual checks on the main mts now and everything seems to be working as expected including the logs however I'll try to do some tests around the breaking changes and will give a heads up to the QAs before test as well

Reply:
Some changes / functionality removed is flagged in the release notes - https://github.com/doctrine/DoctrineORMModule/releases - but they might not impact us.

Reply:
It's been regression tested in dev env, and we didn't see any problems. Marking this as resolved :)